code/game/objects/machinery/sellingpad.dm

```diff
@@ -0,0 +1,65 @@
+/obj/machinery/sellingpad
+	name = "ASRS Bluespace Selling Point"
+	desc = "A bluespace telepad for sending valuble assets, such as valuble minerals and alien corpses. It needs to be wrenched down in a powered area to function."
+	icon = 'icons/obj/machines/telecomms.dmi'
+	icon_state = "broadcaster_off"
+	density = FALSE
+	anchored = FALSE
+	use_power = IDLE_POWER_USE
+	wrenchable = TRUE
+	idle_power_usage = 300
+	active_power_usage = 300
```

Comment:
I would increase this to a higher amount and have each use also just consume a large chunk of power while on cooldown, by flipping the `use_power` to `ACTIVE_POWER_USE` and then back after the cooldown
```suggestion
	active_power_usage = 3000
```